android/.gitignore

```diff
@@ -2,3 +2,4 @@
 google-services.json
 # Keystore
 config/release.keystore
+signing.properties
```

Comment:
The file should be at [the root](https://github.com/nimblehq/kmm-templates/blob/develop/signing.properties). Please skip it from root .gitignore 🙏

Reply:
@luongvo Are you saying that we should put the `signing.properties` on the root path instead inside of `android` package?